api/v1alpha1/olsconfig_types.go

```diff
@@ -103,13 +102,12 @@ type ConversationCacheSpec struct {
 	Type CacheType `json:"type,omitempty"`
 	// +optional
 	Redis RedisSpec `json:"redis,omitempty"`
-	// +optional
-	Memory MemorySpec `json:"memory,omitempty"`
```

Comment:
We removed memory , but still have default type as memory.

Reply:
Removed it. Thank you